Frontend/src/packages/editor/components/buttons/buttonHelpers.ts

```diff
@@ -0,0 +1,25 @@
+import { Editor, BaseEditor } from "slate";
+import { ReactEditor } from "slate-react";
+
+const toggleMark = (
+  editor: BaseEditor & ReactEditor,
+  format: "bold" | "italic" | "underline"
+): void => {
+  const isActive = isMarkActive(editor, format);
+
+  if (isActive) {
+    Editor.removeMark(editor, format);
+  } else {
+    Editor.addMark(editor, format, true);
+  }
+};
+
+const isMarkActive = (
+  editor: BaseEditor & ReactEditor,
```

Comment:
also add comments for what both these functions do, explaining why it is necessary, because toggleMark and isMarkActive is a little vague. Where is the mark coming from? maybe add links to further docs which explains what marks is on slatejs would be beneficial